### ISSUE TITLE
How to get data-placement to include the X axis

### ISSUE THREAD

**Issue by gvocale** [atomiks/tippyjs-react#251]
I want to change the `transform-origin` property depending if the `data-placement` is `top`, `top-start` or `top-end`, and so on, but all the time I manage to get only the following:
`data-placement="top"`
`data-placement="right"`
`data-placement="bottom"`
`data-placement="left"`

How can I get instead the data-placement attribute to be populated with the following?
`data-placement="top-start"`
`data-placement="top"`
`data-placement="top-end"`
`data-placement="right-start"`
`data-placement="right"`
`data-placement="right-end"`
`data-placement="bottom-start"`
`data-placement="bottom"`
`data-placement="bottom-end"`
`data-placement="left-start"`
`data-placement="left"`
`data-placement="left-end"`

<img width="1436" alt="Screen Shot 2020-08-24 at 1 14 04 PM" src="https://user-images.githubusercontent.com/1319291/91080759-c1e10600-e60b-11ea-9409-0ab299c1a68a.png">

**Comment by atomiks** [comment]
That should be the case. Repro?

**Comment by gvocale** [comment]
@atomiks see here https://codesandbox.io/s/react-tippy-data-placement-issue-idwwx?file=/src/App.js

For example all 3 data-placement on the top row are `data-placement="bottom"` without specifying the X axis.

<img width="713" alt="Screen Shot 2020-08-30 at 12 13 55 PM" src="https://user-images.githubusercontent.com/1319291/91665331-6787ef80-eaba-11ea-841f-b517a47b9826.png">

**Comment by atomiks** [comment]
In the Sandbox, no variation placement is specified, you've just mimicked it using a flexbox container (so the `preventOverflow` Popper modifier is working as intended). To get the axis, you need to set a `placement: [base]-[variation]`, e.g. `top-start`.

**Comment by gvocale** [comment]
@atomiks I have tried passing manually `placement='top-start'`. It does work indeed and now i get the data-placement attribute to populate either of the following:
`data-placement='top-start'`
`data-placement='top-end'`
`data-placement='bottom-start'`
`data-placement='bottom-end'`

This way the tooltip doesn't do top-center or bottom-center anymore, which i'd like to be the default. Is there a way to pass instead `placement='top-middle'` `top-center` or else?

Otherwise I won't be able to have the tooltip default to top-center anymore.

I've updated the code sandbox https://codesandbox.io/s/react-tippy-data-placement-issue-idwwx?file=/src/App.js

Thanks!

<img width="727" alt="Screen Shot 2020-08-31 at 12 53 57 AM" src="https://user-images.githubusercontent.com/1319291/91687056-7dca9580-eb24-11ea-9b0d-d26c61d00720.png">
